### DISCUSSION TITLE
how to change the background color of dark mode?

### DISCUSSION THREAD

**Discussion by Daniel-Fei** [zauberzeug/nicegui#2700] [Q&A]
### Question

how to change the background color of dark mode?

**Comment by falkoschindler** [comment]
One possible approach would be to change Quasar's CSS variable for the dark page background:
```py
ui.add_head_html('<style>:root { --q-dark-page: #f00 }</style>')
```

**Reply by Daniel-Fei** [reply]
Thanks a lot for the quick help, it works! I've got another question: Can I use a similar method to set different styles for both normal and dark modes (like the background of the header, font colors, etc.)? Or can I store these styles in separate files and then reference them from the code? Thanks!

**Reply by falkoschindler** [reply]
Yes, you might be able to use such CSS variables. But you would need to find them using the JavaScript developer tools or from the Quasar documentation.
A more natural way to define some kind of default style for certain NiceGUI elements would be to use [default classes](https://nicegui.io/documentation/element#default_classes), [default style](https://nicegui.io/documentation/element#default_style) or [default props](https://nicegui.io/documentation/element#default_props).
Alternatively you can also define some custom Python classes or functions for commonly used UI elements in your project, like we did [here](https://github.com/zauberzeug/nicegui/blob/main/website/style.py) for the NiceGUI website.
